src/main/java/org/gridsuite/cgmes/boundary/server/CgmesBoundaryController.java

```diff
@@ -47,6 +49,15 @@
         return ResponseEntity.ok().contentType(MediaType.APPLICATION_JSON).body(boundaries);
     }
 
+    @GetMapping(value = "/boundaries/last", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Get last boundary", response = List.class)
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "The profile of the required boundary (TP or EQ)")})
```

Comment:
message doesn't really match the purpose of this method

Reply:
Changed